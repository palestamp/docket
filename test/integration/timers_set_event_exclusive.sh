rm ~/.timer
set -o verbose

docket timer new home
docket timer new work

docket timer set home event:onstart:stop work
docket timer set work event:onstart:stop home

docket timer start work
docket timer stat --status-only
docket timer start home
docket timer stat --status-only
