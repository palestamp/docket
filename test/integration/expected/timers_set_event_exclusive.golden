
docket timer new home
docket timer new work

docket timer set home event:onstart:stop work
docket timer set work event:onstart:stop home

docket timer start work
docket timer stat --status-only
*               running   
home            stopped   
work            running   
docket timer start home
docket timer stat --status-only
*               running   
home            running   
work            stopped   
